warehouse/manage/views/__init__.py

```diff
@@ -1861,6 +1862,8 @@ def delete_project(project, request):
 
     remove_project(project, request)
 
+    request.task(tuf.update_metadata).delay(project.id)
```

Comment:
I see these calls a fair bit. Is there anything we could automatically do to reduce the burden of the user?

Reply:
See discussion about completeness above: https://github.com/pypi/warehouse/pull/15815#issuecomment-2182420372

TLDR: This should be improved, but could happen in a follow-up PR.

Reply:
I'll echo this - we don't have many tasks directly called inside views today, rather we listen to the objects being changed via sqlalchemy and trigger tasks that way.

Here's a couple of examples:

https://github.com/pypi/warehouse/blob/e8195e40986d63f9b72926a67eb6acfd43b36932/warehouse/search/__init__.py#L27-L64

https://github.com/pypi/warehouse/blob/e8195e40986d63f9b72926a67eb6acfd43b36932/warehouse/observations/tasks.py#L33-L54